src/sites/image/fotosik.pl.js

```diff
@@ -1,7 +1,9 @@
 _.register({
-  rule: 'https://www.fotosik.pl/*',
+  rule: {
+    host: /^www\.fotosik.pl$/,
```

Comment:
line should be:
host: /^www\\.fotosik\\.pl$/,

Reply:
The code does work without the backslashes though, and passed the checks. I'll add them for consistency anyway since every other rule in the code has them.